bin/index.js

```diff
@@ -213,8 +229,18 @@ module.exports = function(Aquifer, AquiferGitConfig) {
 
       // Commit changes.
       .then(function () {
+        var signature;
+
         Aquifer.console.log('Commit changes...', 'status');
-        var signature = git.Signature['default'](repo);
+
+        if (options.name && options.email) {
```

Comment:
Additionally, I'm not a huge fan of having properties always exist, but have their values be `null`.